docs/source/server-process.md

```diff
@@ -181,9 +181,11 @@ server as raw stream data. This is similar to running a
 [websockify](https://github.com/novnc/websockify) wrapper.
 All other HTTP requests return 405.
 
-#### Callable arguments
+### Callable arguments
```

Comment:
I tweaked the heading levels here because currently this shows up as a subsection of the `raw_socket_proxy` option:

![image](https://github.com/user-attachments/assets/23a2c6a8-409b-484e-82ec-7e8e5d30cd13)